src/components/Amalgamation/AmalgamatingBusinesses.vue

```diff
@@ -235,18 +228,30 @@ export default class AmalgamatingBusinesses extends Mixins(CommonMixin) {
         goodStanding: business.goodStanding
       } as AmalgamatingBusinessIF
 
-      if (!this.amalgamatingBusinesses.find(b => b.identifier === business.identifier)) {
-        this.amalgamatingBusinesses.push(tingBusiness)
+      if (!amalgamatingBusinesses.find((b: any) => b.identifier === business.identifier)) {
+        amalgamatingBusinesses.push(tingBusiness)
+
+        // Set the new amalgamated businesses array in the store.
+        this.setAmalgamatingBusinesses(amalgamatingBusinesses)
```

Comment:
The other way we could have done this is with an action that pushes the new business into the array right in the store, but I think this is OK.